src/main/java/edu/wpi/grip/core/operations/composite/ContoursReport.java

```diff
@@ -0,0 +1,25 @@
+package edu.wpi.grip.core.operations.composite;
+
+import static org.bytedeco.javacpp.opencv_core.*;
+
+/**
+ * The output of {@link FindContourOperation}.  This stores a list of contours (which is basically a list of points) in
+ * OpenCV objects, as well as the width and height of the image that the contours are from, to give context to the
+ * points.
+ */
+public final class ContoursReport {
+    protected int rows = -1, cols = -1;
+    protected MatVector contours = new MatVector();
+
```

Comment:
Ok.  I'm not completely clear on what you mean by variable scoping, but the problem with that is that Jython sees that we have getters but not setters and assumes that `rows` and `cols` aren't supposed to be modified.

Since they *are* supposed to be modified, I'll just add setters.  It's also probably better to do that just in case we decide to move all of the report classes to a separate package.